docker/.gitignore

```diff
@@ -1,6 +1,4 @@
-volumes/db/data/**/*
-!volumes/db/data/.gitkeep
```

Comment:
nested directories are automatically created for volumes, so no need to `.gitkeep` them